models/transfers/celo/erc20/transfers_celo_erc20.sql

```diff
@@ -0,0 +1,94 @@
+{{ config(
+    alias=alias('erc20'),
+    tags=['dunesql'],
+    materialized='incremental',
+    file_format='delta',
+    incremental_strategy='merge',
+    unique_key='unique_transfer_id',
```

Comment:
We may replace the unique_key, to a combination of columns.
i.e.
unique_key = ['type', 'tx_hash', 'evt_index', 'address']

Reply:
Hi @Hosuke, I tried the above, but I keep getting this error msg:

TrinoExternalError(type=EXTERNAL, name=DELTA_LAKE_BAD_WRITE, message="Unable to rewrite Parquet file", query_id=...)

Any other ideas? And btw, thanks for reviewing and all the suggestions so far!

Reply:
I am not sure about the error message as well.